email_backend/templates/feedback_simplified.json

```diff
@@ -0,0 +1,69 @@
+{
+  "meta": {
+    "subject_template": "Feedback Simplified: [${easily_nav}], [${useful_info}], ${lang}, ${route}, ${sha}, ${client_id}"
+  },
+  "easily_nav": {
+    "required": true,
+    "form_type": "radio",
+    "form_label": {
+      "en": "To what extent did you find GC InfoBase easy to navigate? (1=not at all easy and 5=very easy)",
+      "fr": "Dans quelle mesure la navigation est-elle facile dans l’InfoBase du GC? (1=difficile et 5=très facile)"
+    },
+    "value_type": "enums",
+    "enum_values": {
+      "one": {
+        "en": "1",
+        "fr": "1"
+      },
+      "two": {
+        "en": "2",
+        "fr": "2"
+      },
+      "three": {
+        "en": "3",
+        "fr": "3"
+      },
+      "four": {
+        "en": "4",
+        "fr": "4"
+      },
+      "five": {
+        "en": "5",
+        "fr": "5"
+      }
+    },
+    "style": "horizontal"
+  },
+  "useful_info": {
+    "required": true,
+    "form_type": "radio",
+    "form_label": {
+      "en": "To what extent did you find the spending information on GC InfoBase useful? (1=not at all useful and 5=very useful)",
+      "fr": "Dans quelle mesure avez-vous trouvé les renseignements financiers utiles dans l’InfoBase du GC? (1=inutile et 5=très utile)"
```

Comment:
I wonder if we should drop "spending" here and just ask about usefulness of information in general? Might be a bit complicated because the corresponding indicator might specifically refer to spending information though, so we should double check how strictly we have to match that before making changes.

Reply:
General information usefulness would be better IMO. Can you elaborate what you mean on this indicator specifically referring to spending info? Not too familiar with email_backend code but doing a quick `useful_info` search didn't return much "useful info"😏

Reply:
Methodology on this indicator refers to us asking a question specifically about spending information 
![image](https://user-images.githubusercontent.com/11301919/100268293-c70f6300-2f22-11eb-8d35-eee3e817da72.png)

Need to double check with someone whether that restricts us from changing the wording of that question to be more general.

Reply:
ohhhh our own result indicator, okay got it. Who should we ask for that?